vhdl-ts-mode.el

```diff
@@ -1144,6 +1166,8 @@ and the linker to be installed and on PATH."
     (add-hook 'completion-at-point-functions #'vhdl-ts-completion-at-point nil 'local)
     ;; Beautify
     (setq-local vhdl-align-alist vhdl-ts-align-alist)
+    ;; Syntactic overrides
+    (advice-add 'vhdl-in-comment-p :around #'vhdl-ts-in-comment-advice)
```

Comment:
I am wondering, should the advice be added every time the mode is used? Although elisp seems to handle it well and add it only once, so it's not really an issue.

Reply:
You are right. Even though elisp handles it correctly as you mentioned, it makes more sense to place it outside of the major-mode definition function and advice it only once when the package is loaded.

I also added an advice for `vhdl-in-literal` to prevent similar bugs from happening in the future.